chart/values.yaml

```
@@ -667,3 +670,14 @@ obs:
         http: 90011
         # NodePort associated with https port
         https: 90010
+
+openebs-localpv-provisioner:
+  # -- Enable openebs dynamic-localpv provisioner. If disabled replace `openebs-hostpath` storage class
+  # with other viable options.
+  enabled: true
+  # -- If false, openebs NDM sub-chart will not be installed. It's recommended to keep this disabled.
```

```suggestion
  # Enable/disable the openebs NDM sub-chart. It's recommended to keep this disabled.
```
I would not even expose this with --